hw/top_earlgrey/data/clocks.xdc

```diff
@@ -29,3 +29,6 @@ create_clock -add -name clk_spi_out -period 100.00 -waveform {0 5} [get_pin top_
 
 set_clock_groups -group ${clks_10_unbuf} -group ${clks_48_unbuf} -group ${clks_aon_unbuf} -group clk_io_div2 -group clk_io_div4 -group lc_jtag_tck -group rv_jtag_tck -group clk_spi_in -group clk_spi_out -group sys_clk_pin -asynchronous
 
+## Limit delay between target IPs (10/2.5 MHz) and capture clock (100 MHz) for SCA.
+set_max_delay 8 -from ${clks_10_unbuf} -to sys_clk_pin
+set_max_delay 8 -from clk_io_div4 -to sys_clk_pin
```

Comment:
* I'm assuming the 8 is chosen arbitrarily? Can you add a comment for that (and ideally guidance on how it was chosen in case we want to change it).
* The "8" is interpreted as the default time unit, which is typically ns, but can be anything. You should be able to specify "8ns" instead to be sure it's nanoseconds.

Reply:
I agree 8ns would be more clear. 8 was chosen to not over-constrain the path but still confine the resulting trigger delay to within 2 sampling clock cycles (`sys_clk_pin`). I should explain that.